song-server/src/main/java/org/icgc/dcc/song/server/repository/mapper/AnalysisMapper.java

```diff
@@ -39,12 +39,13 @@ public Analysis map(int index, ResultSet r, StatementContext ctx) throws SQLExce
     val study = r.getString("study_id");
     val type = r.getString("type");
     val state = r.getString("state");
+    val info = r.getString("info");
```

Comment:
yes, it should be in a AttributesNames.java class, and then any variable that has the string "study_id" should be changed to reference AttributeNames.STUDY_ID